Toshi/Source/Toshi/TMemory.cpp

```diff
@@ -11,7 +11,7 @@ namespace Toshi
 		TMemManager* TMemManager::s_Class;
 		size_t TMemManager::s_GlobalBlockSize;
 
-		void TMemManager::Initialise(size_t globalBlockSize)
+		void TMemManager::Initialize(size_t globalBlockSize)
```

Comment:
Same here, i wanted to also keep that spelling mistake they did on JPOG

Reply:
oh I didn't even know there was a mistake in JPOG, I thought it was mine mistake lol